pages/common/dotnet-restore.md

```diff
@@ -0,0 +1,28 @@
+# dotnet restore
+
+> Restores the dependencies and tools of a .NET project.
+> More information: <https://docs.microsoft.com/dotnet/core/tools/dotnet-restore>.
+
+- Restore dependencies for a .NET project or solution in the local path:
```

Comment:
Perhaps this will read better?
```suggestion
- Restore dependencies for a .NET project or solution in the current directory:
```

....unless by `local path` you meant the `PATH` environment variable?